app/assets/javascripts/service-worker.js.erb

```diff
@@ -4,7 +4,7 @@ importScripts("<%= "#{Discourse.asset_host}#{Discourse.base_path}/javascripts/wo
 
 workbox.setConfig({
   modulePathPrefix: "<%= "#{Discourse.asset_host}#{Discourse.base_path}/javascripts/workbox" %>",
-  debug: <%= Rails.env.development? %>
```

Comment:
Is there value in setting debug to true? If so maybe we should introduce an ENV for this instead rather than providing no way to override this?

Reply:
It’s safe to set it to always false again in my opinion, as we don’t really had to debug workbox itself in all those years, and if we ever need, it can be a local change for the dev machine.